packages/@n8n/task-runner/src/task-runner.ts

```diff
@@ -92,7 +92,7 @@ export abstract class TaskRunner extends EventEmitter {
 		this.maxConcurrency = opts.maxConcurrency;
 		this.idleTimeout = opts.idleTimeout;
 
-		const wsUrl = `ws://${opts.n8nUri}/runners/_ws?id=${this.id}`;
+		const wsUrl = `ws://${opts.taskBrokerUri}/runners/_ws?id=${this.id}`;
```

Comment:
Do we somehow have an incosistency if the runner expects a scheme-less and port-less URI but the launcher requires them?